web_notify_channel_message/models/res_users.py

```diff
@@ -0,0 +1,14 @@
+from odoo import models
```

Comment:
Hi,
That's not an obligation, but you can too add your license in this python file too.